virttest/qemu_monitor.py

```diff
@@ -1053,6 +1049,16 @@ def set_link(self, name, up):
         :param up: Bool value, True=set up this link, False=Set down this link
         :return: The response to the command
         """
+        # set_link in RHEL5 host use "up|down" instead of "on|off" which is
+        # used in RHEL6 host and Fedora host. So here find out the string
+        # this monitor accept.
+        o = self.cmd("help set_link")
+        try:
+            self.on_str, self.off_str = re.findall("(\w+)\|(\w+)", o)[0]
+        except IndexError:
+            # take a default value if can't get on/off string from monitor.
+            self.on_str, self.off_str = "on", "off"
+
```

Comment:
@luckyh  Here is a new update, it is for human monitor only.  From my checking,  this is the only call for `self.on_str` and `self.off_str`.   Could you help review again ?   Thanks!

Reply:
As we discussed, no need to consider RHEL5, so drop this part directly, only use 'on/off' from now on. Thanks!